pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -79,9 +84,16 @@ func MaybeAddDomainConfig(ctx *zedmanagerContext,
 	for _, sc := range aiConfig.StorageConfigList {
 		// Check that file is verified
 		locationDir := finalDirname + "/" + sc.ImageSha256
-		location, err := locationFromDir(locationDir)
-		if err != nil {
-			return err
+		location := ""
+		err := errors.New("")
```

Comment:
I didn't see this style of err, initliazation. Is empty string nil error ?

Reply:
It is not actually being used. I just need it declared. I think the right way is to declare it as 
         var err errors.Error
          err = nil